src/qml/test/tst_onboarding.qml

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtTest 1.15
+import "../controls"
+import "../pages/onboarding"
+
+Rectangle {
+    id: root
+    width: 640
+    height: 665
+    color: Theme.color.background
+
+    OnboardingWizard {
+        id: onboardingWizard
+        anchors.fill: parent
+    }
+
+    TestCase {
+        id: onboardingFlow
+        when: windowShown
+
+        function test_all_continue_buttons() {
+            verify(onboardingWizard.currentIndex == 0)
+
+            var cover = findChild(onboardingWizard, "onboardingCover")
+            verify(cover, onboardingWizard.currentItem)
+            mouseClick(findChild(cover, "continueButton"))
```

Comment:
What about abstracting a lot of the repetitive action of finding and clicking into a helper function?

helper function looking something like:
```
function clickContinueAndVerify(pageId, expectedIndex) {
    var page = findChild(onboardingWizard, pageId)
    verify(page, onboardingWizard.currentItem)
    mouseClick(findChild(page, "continueButton"))
    verify(onboardingWizard.currentIndex == expectedIndex)
}
```

Then calling like:
```
clickContinueAndVerify("onboardingCover", 1)
wait(200)
clickContinueAndVerify("onboardingStrengthen", 2)
```

Reply:
A lot of tests would involve a repetitive function, so good to think about this now to establish a style for how we write our tests